sql/core/src/test/resources/sql-tests/inputs/show-create-table.sql

```diff
@@ -59,3 +59,8 @@ TBLPROPERTIES ('a' = '1');
 
 SHOW CREATE TABLE tbl;
 DROP TABLE tbl;
+
+-- new real/numeric type
```

Comment:
What we need to prove is the alias works. I think this test is good enough (no need to test it in cast), but we should update the comment. It's not new types, but just alias.

Reply:
OK